python/src/aiconfig/editor/client/src/components/global/ShareButton.tsx

```diff
@@ -1,12 +1,43 @@
-import { Button, Tooltip } from "@mantine/core";
+import { Button, Modal, Tooltip, createStyles, rem } from "@mantine/core";
 import { memo, useState } from "react";
+import { useDisclosure } from "@mantine/hooks";
+import CopyButton from "../CopyButton";
 
 type Props = {
   onShare: () => Promise<string | void>;
 };
 
+const useStyles = createStyles((theme) => ({
+  copyIcon: {
+    alignItems: "flex-end",
+    align: "right",
+    justifyContent: "flex-end",
+    position: "absolute",
+    right: 0,
+  },
+  urlContainer: {
```

Comment:
You could also try using <Container> instead of this wrapping div since it will have padding already